### ISSUE TITLE
Documentation 

### ISSUE THREAD

**Issue by bhagujava** [bobrik/docker-curator#40]
Hi, It would be more use full with documentation

**Comment by karl-tpio** [comment]
Agreed. It took a while to figure out what was going on and how this image differs from the official Dockerfile. The entry point is hard coded and does not seem to like it when you specify the `--config` . flag manually.

here's a short version of how to use this image:

```
$ podman run -it --rm --volume=/etc/curator/jobs/my-job-config.yml:/.curator/curator.yml --volume=/etc/curator/jobs/my-job.yml:/actions.yml bobrik/curator:latest "/actions.yml"


{"@timestamp": "2019-12-09T21:13:49.702Z", "function": "run", "linenum": 148, "loglevel": "INFO", "message": "Preparing Action ID: 1, \"snapshot\"", "name": "curator.cli"}
...  etc
```

Once i stopped trying to set the `--config` flag manually (was being told `Error: no such option: --config /curator.yml`) and just used the path that the image expects:`/.curator/curator.yml` things more or less worked.
